build-system/tasks/report-test-status.js

```diff
@@ -32,7 +32,7 @@ const IS_LOCAL_CHANGES = !!argv.local_changes;
 const IS_DIST = !!argv.compiled;
 
 const TEST_TYPE_SUBTYPES = new Map([
-  ['integration', ['local', 'minified']],
+  ['integration', ['local', 'minified', 'sxg']],
```

Comment:
Thanks for adding this. There are a couple more changes needed to get this to work:

a. Add a new const called `IS_SXG` above `IS_DIST` (line 32), based on `argv.sxg`.
b. In `inferTestType()`, add a new `else if` block before `IS_DIST` which returns `${type}/sxg`.

Reply:
Done.